rust/operator-binary/src/druid_controller.rs

```diff
@@ -689,7 +689,8 @@ fn build_rolegroup_config_map(
     druid_tls_security: &DruidTlsSecurity,
     druid_auth_config: &Option<DruidAuthenticationConfig>,
 ) -> Result<ConfigMap> {
-    let role = DruidRole::from_str(&rolegroup.role).unwrap();
+    let druid_role = DruidRole::from_str(&rolegroup.role).unwrap();
```

Comment:
Nitpick: use an error? I know it cannot happen but i always nervous seeing unwraps :D
```
    #[snafu(display("could not parse druid role: {role:?}"))]
    DruidRoleParseFailed { role: String }
```

Reply:
Sure thing! 5e6b631